sdk/eventhub/event-processor-host/test/README.md

```diff
@@ -0,0 +1,63 @@
+# Testing
+
+To run our tests, first do a full update of the project by running `rush update`, then rebuild either this package, with `rush build -t @azure/event-processor-host` or all packages with `rush build`, then make sure to build the tests with `rush build:test`. After that, you can either run our unit tests, which won't reach to live services, and our integration tests, which will indeed require existing, reachable resources.
+
+## Run all tests
+
+Run `rushx test:node` or `rushx test:browser` from your terminal to run all the tests under the `./test` folder
+
+## Run all tests in a single test suite
+
+Append the `.only` on the `describe` method corresponding to the test suite.
+
+Then run `rushx test:node` or `rushx test:browser` from your terminal.
+
+## Run a single test
+
+Append the `.only` on the `it` method corresponding to the test.
+
+Then run `rushx test:node` or `rushx test:browser` from your terminal.
+
+## Debug tests using Visual Studio Code
+
+- Select the debug configuration `Debug Unit Tests` from the drop down in the debug viewlet.
+- Add breakpoints in the test you want to debug. Tip: It is helpful to append the `.only` on the test method if you want only that method to be run.
+- Start Debugging
+
+## Unit tests
+
+To run unit tests, go to the root of this library and run `rushx unit-test`. This will run tests in a purely nodejs environment, and in a browser environment using [Karma](https://karma-runner.github.io/latest/index.html). These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses. You won't need to have any environment configuration, nor internet access, for these tests to run.
```

Comment:
```suggestion
To run unit tests, go to the root of this library and run `rushx unit-test`. This will run tests in NodeJS and in a browser (through [Karma](https://karma-runner.github.io/latest/index.html)). These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses. You won't need to have any environment configuration, nor internet access, for these tests to run.
```